kong/reports.lua

```diff
@@ -41,6 +41,8 @@ do
     _buffer[#_buffer + 1] = k .. "=" .. v
   end
 
+  _buffer[#_buffer + 1] = "prd=kong"
```

Comment:
If we intend to customize the value in other places (e.g., Enterprise), could we define the value in `kong.constants` (or obtain from `kong.meta`)?